src/components/ContactSalesForm/index.tsx

```diff
@@ -148,9 +149,9 @@ export default function ContactSalesForm() {
                 buttonClass={styles.phoneDropdown}
                 country={'us'}
                 value={phone}
-                onChange={() => addPhone()}
+                onChange={(phone) => handlePhoneChange(phone)}
```

Comment:
nit: it can be just this, since signature is the sasme:

```suggestion
                onChange={handlePhoneChange}
```